sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/crypto/_internal/symmetric_key.py

```diff
@@ -56,6 +56,14 @@ def __init__(self, kid=None, key_bytes=None, key_size=None):
 
         self._key = key_bytes
 
+    def is_private_key(self):
+        return True
+
+    @classmethod
+    def from_jwk(cls, jwk):
+        # RFC 7517 specifies 'k' be url-safe base64 encoded
+        return cls(kid=jwk.kid, key_bytes=base64.urlsafe_b64decode(jwk.k))
```

Comment:
I believe that jwk.k will be bytes just like all the other key specific attributes on the JsonWebKey model that we expose. We shouldn't expect this to be encoded.